client/logs_test.go

```diff
@@ -20,12 +20,17 @@ import (
 	"fmt"
 	"net/http"
 	"net/url"
+	"strings"
 
 	"gopkg.in/check.v1"
 
 	"github.com/canonical/pebble/client"
 )
 
+const (
+	maxMessageSize = 4 * 1024
```

Comment:
To reduce the scope, I'd probably put this as a local const inside the test function (that works fine in Go). Also, you can use a one-liner `const maxMessageSize = 4 * 1024`.